src/parsers/forecast_parser.jl

```diff
@@ -293,3 +293,42 @@ function parse_json(filename,device_names)
     return Devices
 end
 =#
+
+"""
+    remove_all_forecasts(sys::System, T::Type{T})::Vector{T} where T <: Forecast
+
+Remove all forecast objects of concrete type T from a System
+"""
+function remove_all_forecasts!(sys::System, ForecastType::Type{T})where T <: Forecast
+
+    ts = get_forecast_initial_times(sys)
+
+    for t in ts
+        f = get_forecasts(ForecastType, sys, t)
+        foreach(x -> remove_forecast!(sys,x),f)
+    end
```

Comment:
missing return

Reply:
Do we need that on a function that doesn't return anything?

Reply:
http://www.juliaopt.org/JuMP.jl/dev/style/#Return-statements-1

Reply:
Good point.  I forgot about that.  I don't love that rule and have been violating it...  :(

You would never do that in C or C++ unless you return early.